Documentation/ContentObjects/Hmenu/Tmenu/Index.rst

```diff
@@ -287,7 +287,7 @@ Properties
 
     ..  confval:: target
         :name: menu-common-properties-target
-        :type: :ref:`data-type-target`
+        :type: string
```

Comment:
In rendered version this value is not linked to ref anymore. So no one will now about the special value "content", right?

Reply:
There is not really any special value here. the "target" is just a string, the semantic meaning of it is a feature of HTML, not TypoScript